### PR TITLE
Helm chart to create the superset default charts

### PR DESCRIPTION
**Pull request by manjudr**

### CODE REVIEW

File: terraform/aws/main.tf [hunk 1]
```diff
@@ -89,6 +89,13 @@ module "superset" {
   postgresql_service_name           = module.postgresql.postgresql_service_name
 }
 
+module "superset_charts" {
+  source                            = "../modules/helm/superset-charts"
+  env                               = var.env
```
Comment (manjudr) [inline terraform/aws/main.tf:94]: Please fix the indentations

File: terraform/modules/helm/superset-charts/superset-charts-helm-chart/templates/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: superset-charts
+  namespace: {{ .Values.namespace }}
+spec:
+  restartPolicy: Never  # Set restart policy for the pod to "Never"
+  containers:
+    - name: superset
+      image: "python:3.9"
```
Comment (manjudr) [inline terraform/modules/helm/superset-charts/superset-charts-helm-chart/templates/deployment.yaml:10]: Have we verified the vulnerabilities?

**Comment by manjudr** [comment]
@shiva-rakshith  - Please verify the vulnerabilities and share the report here

**Comment by manjudr** [comment]
Where is the ingestion spec to submit the failed events?

**Comment by shiva-rakshith** [comment]
> Where is the ingestion spec to submit the failed events?

https://github.com/Sanketika-Obsrv/obsrv-automation/pull/89

**Comment by ravismula** [comment]
Closing it as its very old PR. Changes will be raised in a new PR

**Pull request closed without merging**